src/device/intrinsics/wmma.jl

```diff
@@ -15,7 +15,8 @@ const map_ptx_to_jl_array = Dict(
                                  "s8"  => Int8,
                                  "s32" => Int32,
                                  "f16" => Float16,
-                                 "f32" => Float32
+                                 "f32" => Float32,
+                                 "f64" => Float64
```

Comment:
Unrelated changes?

Reply:
I added intrinsics calls for WMMA with directed rounding modes

Reply:
Can you keep that to a separate PR? We also currently don't support Float64 WMMA, see https://github.com/JuliaGPU/CUDA.jl/pull/1426.